plugins/squads/src/components/SquadEntityPage/EntityContextMenu/EntityContextMenu.test.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render, fireEvent } from '@testing-library/react';
+import * as React from 'react';
+import { act } from 'react-dom/test-utils';
+import { EntityContextMenu } from './EntityContextMenu';
+
+describe('ComponentContextMenu', () => {
+  it('should call onUnregisterEntity on button click', async () => {
+    await act(async () => {
```

Comment:
Wrapping the entire test in an `act` doesn't do much, does it? ^_>

Reply:
I think it's just the `fireEvent` you'd need to wrap. I also think it might be possible to do `await fireEvent` IIRC.

Or just the part after `render` if nothing else works 🤓

Reply:
Not sure what part of this that needs the act at all, hmm. fireEvent should not need wrapping or awaiting, it's synchronous. For even more realism, one can use userEvent instead https://github.com/testing-library/user-event#clickelement-eventinit-options

Reply:
@freben I think it depends on what the click handler does. If it goes and does something async it will throw `act` errors.

Reply:
https://medium.com/@davidwcai/react-testing-library-and-the-not-wrapped-in-act-errors-491a5629193b

Reply:
TBH I havn't looked at the tests at all. Just copied them from the catalog plugin 😅